src/lib/containers/drug_upload_tool/DrugUploadTool.css

```diff
@@ -0,0 +1,164 @@
+/* MODAL OVERWRITE! */
```

Comment:
wo we need to track .css files at all?

Reply:
we shouldn't have to, its been on the backlog to remove that, theres one thing in the build script that has to get changed so we can ignore those files. I think the change is to build the css and then the project but right now its the opposite.